src/components/ArticlesHome.js

```diff
@@ -7,22 +7,48 @@ import Moment from "moment";
 
 const ArticlesHome = () => {
   const [articlesData, setarticlesData] = useState([]);
-  //const [filteredState, setFilteredState] = useState("");
+  const [order, setOrder] = useState("DESC");
+  const [sortBy, setsortBy] = useState("votes");
 
   useEffect(() => {
-    fetchArticlesData().then((articles) => {
+    fetchArticlesData(sortBy, order).then((articles) => {
       setarticlesData(articles);
     });
-  }, []);
+  }, [sortBy, order]);
 
   const navigate = useNavigate();
 
   const handleClick = (event) => {
     navigate(`/article/${event}`);
   };
 
+  const handleChange = (event) => {
+    setOrder(event.target.value);
+  };
+
+  const handleSortByChange = (event) => {
+    setsortBy(event.target.value);
+    // orderDisplayed = event.target.value;
+  };
+
   return (
     <>
+      <nav className="Filters">
+        <select class="select" value={sortBy} onChange={handleSortByChange}>
+          <option value="created_at">date added</option>
+          <option value="author">author</option>
+          <option value="title">title</option>
+          <option value="body">body</option>
+          <option value="topic">topic</option>
```

Comment:
If you want to sort on topic later, remove it for now and come back at another time.